### PR TITLE
Fixed Timeline pixelPerSecond and Export SSML URL with wrong "begin" values

### PR DESCRIPTION
**Pull request by davidcommarmond**
The pixelPerSecond value was set to 100, however the ticker interval was being calculated with 40px interval, thus rendering incorrect "Perfect second" pixel values.
I've changed that to render every 50px instead of every 40px.

It was also rendering the ticker timeline with 0.5s advance, which didn't reflect properly in the Export SSML being generated, creating confusion. I fixed that as well.

And lastly, the begin values were trying to be calculated relative to the previous block, this causing the URL Import to bug when trying to parse the values. I changed that to provide absolute begin values instead of relative, which fixed the URL importer.

### CODE REVIEW

File: src/ssml-types/audio-library.ts [hunk 1]
```diff
@@ -36,7 +36,7 @@ const getSsml = (data: LibraryData) => {
 export default {
   getTimelineHtml: (data: LibraryData) => {
     return `<strong class="audio-description">
-            <img src="./images/library.png" />
+            <i class="material-icons">library_music</i>
```
Comment (Fleker) [inline src/ssml-types/audio-library.ts:39]: Would you mind reverting this change and moving to a separate PR? That would make this one a bit more contained.

File: src/modules/ssml-timeline.ts [hunk 2]
```diff
@@ -336,14 +336,20 @@ export class SsmlTimeline extends PolymerElement {
...
-      tickerHtml += '<div part=\'ticker-tick\' class=\'ticker-tick\'>'
-      if (i % this.pixelsPerSecond === 0) {
-        // Perfect second
-        tickerHtml += `${i / this.pixelsPerSecond}s`;
-      } else {
-        tickerHtml += `|`
+    for (let i = 0; i < max; i += 50) {
+      // Put something every 50 pixels
+      tickerHtml += '<div part=\'ticker-tick\' class=\'ticker-tick\'>';
+      if(i == 0){
```
Comment (Fleker) [inline src/modules/ssml-timeline.ts:342]: Can you revise the formatting here, using 3 equal signs for equality and spaces between `if`, `else`, and brackets?

**Review by Fleker — CHANGES_REQUESTED** [review]

File: src/ssml-types/audio-library.ts (hunk 1 above)
Comment (davidcommarmond) [reply src/ssml-types/audio-library.ts]: @Fleker ok reverted that commit and will PR from a branch https://github.com/GWS-Technologies-LTD/nightingale-ssml-editor/tree/google-icons

File: src/modules/ssml-timeline.ts (hunk 2 above)
Comment (davidcommarmond) [reply src/modules/ssml-timeline.ts]: @Fleker ok done

**Comment by davidcommarmond** [comment]
@Fleker I just noticed that my google-cla agreement just got approved now.
Should I cancel this PR and do a new one so the cla/google check can pass ? Or you can do it from your side ?

Thanks

**Comment by davidcommarmond** [comment]
@Fleker ok CLA passed
Thanks

**Review by Fleker — APPROVED** [review]
Thanks for picking out this fix! Looks good to me!
